packages/libs/react-ui/src/components/Notification/Notification.css.ts

```diff
@@ -51,7 +51,7 @@ export const cardColorVariants = styleVariants(colorVariants, (color) => {
 
 export const expandVariants = styleVariants({
   true: [sprinkles({ width: '100%', maxWidth: '100%' })],
-  false: [sprinkles({ width: 'max-content', maxWidth: 'maxContent' })],
+  false: [sprinkles({ width: 'max-content', maxWidth: '$maxContent' })],
```

Comment:
why is width not `$maxContent`?

Reply:
This PR doesn't change any existing styling. The notification's component was implemented to be the width of it's max-content, however I believe this will change in the future when the notification's component is refactored with the new UX designs in figma

Reply:
Also `$maxContent` is a custom token that is 42.5rem / 680px while `max-content` is the native css property value. Maybe I can update the token to be `$maxContentWidth`?

Reply:
yes, `$maxContentWidth` avoid confusion. else it resonates same as `max-content`